fetching/tarball-fetcher/src/index.ts

```diff
@@ -39,7 +41,16 @@ export function createTarballFetcher (
     offline?: boolean
   }
 ): TarballFetchers {
-  const download = createDownloader(fetchFromRegistry, {
+  const workerPool = new WorkerPool({
+    id: 'tarball',
+    maxWorkers: 16,
```

Comment:
The maximum safe upper bound for this value is `os.cpus().length - 1`, since each thread other than the main thread will be saturated during parse/extract; we are unlikely to see a performance gain from this change if the host machine has 2 or fewer CPUs.

Reply:
Done